internal/module/module.go

```diff
@@ -61,11 +53,60 @@ func parseModuleFile(repoRoot string) (*build.File, error) {
 // Collects the mapping of module names (e.g. "rules_go") to user-configured apparent names (e.g.
 // "my_rules_go"). See https://bazel.build/external/module#repository_names_and_strict_deps for more
 // information on apparent names.
-func collectApparentNames(m *build.File) map[string]string {
+func collectApparentNames(repoRoot, relPath string) (map[string]string, error) {
```

Comment:
We should probably add a test for this function

Reply:
I added a test and it also found a bug - I didn't know that `Rule` has an `ImplicitName`.